meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/tegra-binaries/tegra-redundant-boot/nv_update_verifier.init

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh
```

Comment:
I think we can remove this since it appears to match the one at https://github.com/OE4T/meta-tegra/tree/kirkstone-l4t-r32.7.x/recipes-bsp/tegra-binaries/tegra-redundant-boot ?

Reply:
Nevermind, we need it because we are calling nv_update_verifier instead of nv_update_engine --verify.  Might be good to upstream a change which allows the recipe to override the verify utility instead.